lib/jobs/create-default-pollers.js

```diff
@@ -83,8 +83,9 @@ function createDefaultPollersJobFactory(
      */
     CreateDefaultPollersJob.prototype.createUcsPoller = function(nodeIds, poller){
         var self = this;
-        return Promise.each(nodeIds, function(nodeId) {
-            return self._createWorkitem(nodeId, poller, "ucs-obm-service");
+        return Promise.map(nodeIds, function(nodeId) {
+            var clonedPoller = _.cloneDeep(poller);
```

Comment:
This change is also required for CreateDefaultPollersJob.prototype.createWsmanPoller and CreateDefaultPollersJob.prototype.createRedfishPoller, suggest you to move it to _createWorkitem.

Reply:
I have considered add the same logic into Wsman and Redfish, but I am not sure if this bigger scope change will cause unexpected issues for these two pollers.
OK, I will move new logic into _createWorkitem method and test it.

Reply:
fixed.